app/konnect/mesh-manager/service-mesh.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configure a Mesh Global Control Plane with the Kubernetes demo app
+content_type: tutorial
+beta: true
+---
+
+Using Mesh Manager, you can create {{site.mesh_product_name}} global control planes to manage your {{site.konnect_saas}} mesh services. This guide explains how to configure a global control plane with {{site.mesh_product_name}} and the Kubernetes demo app. You will use Kubernetes to install various service mesh components.
+
+{:.important}
+> **Important:** Mesh zones are priced based on consumption. For more information about the pricing and consumption of zones, see Kong's [Pricing](https://konghq.com/pricing) page.
+
+## Prerequisites
+
+* [A Kubernetes cluster with load balancer service capabilities](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types)
+* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [Download the latest version of {{site.mesh_product_name}}](/mesh/latest/production/install-kumactl/)
+
+## Create a global control plane in {{site.konnect_short_name}}
+ 
+1. From the left navigation menu in {{site.konnect_short_name}}, open {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager).
+1. Click **New Global Control Plane**.
+1. Enter "example-cp" in the **Name** field and click **Save**.
+
+You now have a {{site.mesh_product_name}} global control plane. This control plane won't have any functionality until you connect a zone to it.
+
+## Create a zone in the global control plane
+
+After creating the global control plane, you must add a zone to that control plane. Adding a zone allows you to manage services added to that zone and send and receive configuration changes to the zone. 
+
+1. Select the `example-cp` control plane you just created and then click **Create Zone**. 
+1. Enter "zone-1" in the **Name** field for the new zone, and then click **Create Zone & generate token**. 
+    
+    {:.note}
+    > **Note:** The zone name must consist of lower case alphanumeric characters or `-`. It must also start and end with an alphanumeric character.
+1. Follow the instructions that display to set up Helm and a secret token. 
+    Once {{site.konnect_short_name}} finds the newly created zone, it will display it. 
```

Comment:
```suggestion
    {{site.konnect_short_name}} will automatically start looking for the zone, once {{site.konnect_short_name}} finds the zone, it will display it. 
```
I think the piece about it looking for it is important, but feel free to ignore this.